public/src/core/pipes/shorten.pipe.ts

```diff
@@ -7,7 +7,10 @@ import { Pipe, PipeTransform } from '@angular/core';
  * @class ShortenPipe
  * @implements {PipeTransform}
  */
-@Pipe({ name: 'shorten' })
```

Comment:
Do we use this?

Reply:
Like above, will check this, it certainly is used somewhere I noticed before, not sure admin or public.

Reply:
It's been used in 'public' page.